gradle.properties

```diff
@@ -1,2 +1,3 @@
 android.useAndroidX=true
-org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
\ No newline at end of file
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
+gnsp.disableApplyOnlyOnRootProjectEnforcement=true
```

Comment:
Why was this added?

Reply:
>* What went wrong:
An exception occurred applying plugin request [id: 'io.codearte.nexus-staging']
> Failed to apply plugin 'io.codearte.nexus-staging'.
   > Nexus staging plugin should ONLY be applied on the ROOT project in a build. See https://github.com/Codearte/gradle-nexus-staging-plugin/issues/47 for explanation. If you really know what you are doing it can be overridden with setting 'gnsp.disableApplyOnlyOnRootProjectEnforcement' property.
   
The problem is, we have multi-project setup, and I cannot take the whole lib' `publishing` block to the root